styles.css

```diff
@@ -483,6 +626,67 @@ nav li a img {
   color: #fff;
 }
 
+.popupbackgroundmobile {
+  width: 100vw;
+  height: 100vh;
+  display: none;
+  z-index: 10;
+  position: fixed;
+  top: 0;
+  background-color: #396df277;
+  backdrop-filter: blur(5px);
+}
+
+#demo-mobile {
+  width: 90%;
+  height: 580px;
+  display: none;
+  position: absolute;
+  top: 1%;
+  z-index: 20;
+  background-color: #fff;
+  overflow: scroll;
+  padding: 10px 20px;
+  margin-left: 18px;
+  margin-top: 20px;
+  transition: width 4s;
+}
```

Comment:
- [ ] Kindly make sure to adjust the right of the popup card on the mobile version in order to meet the provided Figma design as much as possible. :wink: 

   - ### YOUR DESIGN
   
       ![image](https://user-images.githubusercontent.com/87197412/217765741-394c1828-f499-47b9-92aa-bcabb495089b.png)

   - ### FIGMA DESIGN
   
       ![image](https://user-images.githubusercontent.com/87197412/217765599-6f4f455d-18a9-47a4-b737-91f4fff8bf68.png)

Reply:
![image](https://user-images.githubusercontent.com/117091923/217860280-be931b75-8a7f-4859-aecb-a4dcc9a0b0fe.png)